app/views/embed/topics.html.erb

```diff
@@ -7,7 +7,7 @@
       </button>
     <%- end %>
   <%- end %>
-  <div class='topics-list' data-embed-state='loaded' <%- if @embed_id %>data-embed-id="<%= @embed_id %>"<%- end %>>
+  <div class='topics-list <%= @embed_class %>' data-embed-state='loaded' <%- if @embed_id %>data-embed-id="<%= @embed_id %>"<%- end %>>
```

Comment:
I assume this is sanitized? It might do us good to validate the format of the string in the controller, /[a-zA-Z0-9\-]+/ or similar?